utilities/cli/holohub.py

```diff
@@ -314,6 +314,11 @@ def _create_parser(self) -> argparse.ArgumentParser:
         setup.add_argument(
             "--dryrun", action="store_true", help="Print commands without executing them"
         )
+        setup.add_argument(
+            "--scripts",
+            action="append",
+            help="Named dependency installation scripts to run. Can be specified multiple times. Searches in HOLOHUB_SETUP_SCRIPTS_DIR directory.",
```

Comment:
"Searches in HOLOHUB_SETUP_SCRIPTS_DIR directory." is more relevant to the maintainer of the repo and might not be very informative for the users. 
Instead, can we list the name of scripts in HOLOHUB_SETUP_SCRIPTS_DIR? Thanks